docs/A-Introduction/information.md

```diff
@@ -90,10 +103,17 @@ For example, 4 Gigabytes of RAM
 * starts at a low address of 0x00000000
 * ends at a high address of 0xFFFFFFFF
 
-![](/img/image21.png)
+<span className="mdImg" align="center" alt="An unique byte address image">
+
+![Bytes](../../static/img/image21.png)
+
+</span>
+
+:::note 
 
->**Note** that each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
+Each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
 
+:::
 
 ### Sets of Bytes
 
```

Comment:
I would suggest escaping any `*` character that represents multiplication (line 122-128), as some of them will get converted into `_` after being formatted with Prettier. See #56 for more info.

Reply:
> I would suggest escaping any `*` character that represents multiplication (line 122-128), as some of them will get converted into `_` after being formatted with Prettier. See #56 for more info.

Did you mean the multiplication from line 115-122?

Reply:
I have modified them, thx